src/docs/framework-integration/angular.md

```diff
@@ -39,211 +43,179 @@ top-most-directory/
+#### Creating an Angular Component Library
 
-In your CLI tool of choice, install Angular’s CLI tools:
+> If you already have an Angular component library, skip this section.
 
-```bash
-npm install -g @angular/cli
-```
+The first time you want to create the component wrappers, you will need to have an Angular library package to write to.
 
-Next, you will create your Angular workspace and project. Ideally, this should be placed next to your Stencil project, so a monorepo structure is encouraged. 
+Using Angular's CLI, generate a workspace and a library for your Angular component wrappers:
 
 ```bash
-ng new angular-workspace --no-create-application
+npx -p @angular/cli ng new angular-workspace --no-create-application
 cd angular-workspace
-ng generate library component-library
+npx -p @angular/cli ng generate library component-library
 ```
 
-### Create a Stencil Component Library
+#### Creating a Stencil Component Library
 
-Your Stencil library will be the location that you write your web components. 
+> If you already have a Stencil component library, skip this section.
 
 ```bash
 npm init stencil components stencil-library
 ```
 
-### Install the Angular Output Target in your Stencil Component Library
+### Adding Angular Output Target
 
-Now that the project structure is set up, we can install the Angular Output Target package. This package contains the Angular wrapper function that we will use to generate our Angular wrapped components. To install the Angular Output Target package, run the following command in your Stencil project directory
+Install the `@stencil/angular-output-target` dependency to your Stencil component library package.
 
-`npm i @stencil/angular-output-target`
-
-or
-
-`yarn add @stencil/angular-output-target`
+```bash
+npm install @stencil/angular-output-target --save-dev
+```
 
-### Add the Angular Wrapper Function to your Stencil Component Library
-With the Angular Output Target package installed, we can now configure our Stencil Component Library to build our Angular wrapped components. In the `stencil.config.ts` file of your Stencil component library, add the Angular wrapper function. 
+In your project's `stencil.config.ts`, add the `angularOutputTarget` configuration to the `outputTargets` array:
 
-Within your Stencil’s config file, you can import the Angular wrapper function for use within the outputTargets array. If you copy and paste this, ensure you update the “my-workspace” and “my-lib” to coincide with what you named your component library, as well as to update the `componentCorePackage` to the name of your Stencil component library.
+```ts
+import { angularOutputTarget } from '@stencil/angular-output-target';
 
-```tsx
-import { angularOutputTarget as angular } from '@stencil/angular-output-target';
 export const config: Config = {
-  namespace: 'demo',
+  namespace: '',
```

Comment:
Updated to be `component-library` to match the other usage examples in the API documentation. I think we should likely sweep these and make them align with the default namespace in the future. Thoughts?

Reply:
Agreed. I think the tricky part is that our Angular and React documentation tries to be both accommodating existing projects (e.g. we say things like "If you already have a Stencil component library, skip this section") as well as being tutorial-like for folks that don't.  In the case of the former, the namespace will be set already (likely to whatever the top level directory containing the stencil library is named), whereas the latter we have "stencil-library".  I don't think this problem is unsolvable by any means, just something we'll want to be aware of moving forward.

Reply:
Actually, we may want to call this 'stencil-library', because that namespace is populated (as 'stencil-library') by the command we provide in the previous section:
```
npm init stencil components stencil-library
```